frontend/packages/dev-console/src/components/import/import-validation-utils.ts

```diff
@@ -39,6 +39,42 @@ export const validationSchema = yup.object().shape({
         message: 'This field cannot be empty.',
       }),
   }),
+  serverless: yup.object().shape({
+    enabled: yup.boolean(),
+    scaling: yup.object().when('enabled', {
+      is: true,
+      then: yup.object({
+        minpods: yup
+          .number()
+          .integer('Pods must be an Integer.')
```

Comment:
Should this say `Min Pods must be an Integer.`? 
cc @parvathyvr

Reply:
@invincibleJai  here as Christian has pointed,it should be 'Min Pods must be an Integer'.I confirmed with Steve in the Devtools IXD Stakeholder's meeting